README.md

```diff
@@ -73,6 +73,53 @@ with alerter.simulation_alert("super dooper sim"):
 which will send "BigSims: super dooper sim has completed without error." via slack once
 `run_simulation()` has completed.
 
+## CLI Usage
+
+    Usage: simulert [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+      -e, --email                     attach email handler
+      -s, --slack                     attach slack handler
+      --simulertSlackToken TEXT       the token for the slack-bot used to send
```

Comment:
I don't think these need the `simulert` prefix.

Reply:
I tried to keep consistency with the environment variable but I agree, the `simulert` prefix is a little redundant here